app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -225,9 +235,16 @@ class HomePresenter(
 
     private fun handleUserTokensLoaded(userTokens: List<Token.Active>) {
         launch {
+            val ethereumTokens = try {
+                ethereumRepository.loadWalletTokens().map { TokenConverter.ethMetadataToToken(it) }
```

Comment:
` ethereumRepository.loadWalletTokens()` should return domain model.
Call `TokenConverter.ethMetadataToToken(it)` in repository

Reply:
we can not do that because TokenConverter on app level and repository on ethereumKit level and it does not know anything about TokenConverter @1sakov